internal/backend/storage/gitfs/commands.go

```diff
@@ -0,0 +1,51 @@
+package gitfs
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/gopasspw/gopass/internal/out"
+	"github.com/gopasspw/gopass/pkg/ctxutil"
+	"github.com/urfave/cli/v2"
+)
+
+func (l loader) Commands(i func(*cli.Context) error, s func(string) (string, error)) []*cli.Command {
+	return []*cli.Command{
+		{
+			Name:  "git",
+			Usage: "Run a git command inside a password store",
+			Description: "" +
+				"If the password store is a git repository, execute a git command " +
+				"specified by git-command-args.",
```

Comment:
BTW, `git-command-args` is mentioned nowhere in the output.

Maybe change the `Usage` part to contain it: 
```
gopass git store git-command-args...
```